tests/Molecule_test.cpp

```diff
@@ -96,17 +96,17 @@ BOOST_AUTO_TEST_CASE ( parseXYZFile ) {
     BOOST_REQUIRE_THROW(GQCP::Molecule::Readxyz("this is a nonsense data path"), std::runtime_error);
 
     // Make sure we get an error when a path with a wrong extension is given
-    BOOST_REQUIRE_THROW(GQCP::Molecule::Readxyz("../tests/ref_data/nuclear.data"), std::runtime_error);
+    BOOST_REQUIRE_THROW(GQCP::Molecule::Readxyz("ref_data/nuclear.data"), std::runtime_error);
```

Comment:
I see now that I made a mistake in a previous commit. This should be `data/small_vector.data`